lib/paths.cf

```diff
@@ -570,6 +586,32 @@ bundle common paths
 
       "path[shadow]"       string => "/etc/security/passwd";
 
+    termux::
+      # some adjustments for the basics
+      "path[tar]"           string => "/usr/bin/tar";
+      "path[true]"          string => "/usr/bin/true";
+      "path[false]"         string => "/usr/bin/false";
+      "path[cat]"           string => "/usr/bin/cat";
+      "path[sysctl]"        string => "/usr/bin/sysctl";
```

Comment:
Curiosu if these really need to be over-ridden from debian or linux.

Reply:
linux has:

```
      "path[tar]"           string => "/bin/tar";
      "path[true]"          string => "/bin/true";
      "path[false]"         string => "/bin/false";
```

So that's a yes for those and the others aren't handled otherwise and would default to just /bin so must be over-ridden.

Termux environment doesn't define debian even though it uses apt. :)